.travis.yml

```diff
@@ -20,11 +25,15 @@ matrix:
         - pip install --only-binary=numpy,scipy -r requirements.txt
       script:
         - nosetests
-deploy:
-  provider: script
-  script: bash cd/deploy.sh
-  on:
-    branch: master
+      deploy:
```

Comment:
How does this jive with "we might not be able to use the backed-in pypi deployment script to travis because the setup.py is not in the root directory."

Reply:
It doesn't. Yesterday, I didn't consider having two different deploy statements: one for the scala code and one for python. That lets me execute the deployment from the two different directories.

Reply:
We'll see if I need to add a:

```
before_deploy:
  - cd python
```

to the python build route. I don't think we do, but the deployment not finding setup.py is the error that would not surprise me.